sos/report/plugins/storcli64.py

```diff
@@ -0,0 +1,46 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+import os
+
+
+class StorCLI64(Plugin, IndependentPlugin):
+
+    short_desc = 'LSI MegaRAID devices'
+
+    plugin_name = 'storcli64'
```

Comment:
So a while back we updated the `megacli` plugin, and ran into feedback that `storcli` was abandoned even though it was originally intended to _replace_ `megacli`.

Has that changed? Is there actual use/development going on for `storcli`?

If so, I'd suggest a slight modification to the `short_desc`. The `megacli` plugin currently has the same description string, which might confuse users to see two different plugins with the same description.

Reply:
Working with the vendor (LSI) recommendation is to use storcli64.
`LSI has created a tool to help manage RAID devices called StorCLI (previously MegaCLI).`

What change in the description would you recommend?